cabal.project

```diff
@@ -95,7 +95,7 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/maestro-org/haskell-sdk
-  tag: deb1e133cd7de989e9e85c2b44e9361f52e93776
+  tag: 541bc5a447039d12e89748a44fd897674e693856
```

Comment:
```suggestion
  tag: 3c4efccb00df8e9fe0313dfb5228e294caa1807d
```

This is from incorporating your changes into this PR: https://github.com/maestro-org/haskell-sdk/pull/33.